### PR TITLE
Adds a new rifle type to marines-Skirmish Rifle

### PR DESCRIPTION
**Pull request by Pariah919**
## About The Pull Request
Adds the T-21 Skirmish rifle, filling the void between a battle rifle and an assault rifle.
It does 30/10/1.5, fires Heavy bullets and it's packet is 125 rounds
image

## Why It's Good For The Game
Adds a new class of rifle which bridges a gap we really don't have right now, and hopefully opens up new styles of play. 
It also looks and sounds great.

## Changelog
:cl:
add: Added new Skirmish Rifle (T-21) to marines, it bridges the gap between a Battle rifle and a Assault rifle. It is 0.2- 30/10/1.5. Aim mode stats are +0.15 and 2.5.
/:cl:

### CODE REVIEW

**Review by Kuro020 — APPROVED** [review]
the blandest gun in the west, but it's ok

**Comment by David-Stormwell** [comment]
> the blandest gun in the west, but it's ok

go harder on him, we both know the tan and middle section which looks like it was ripped from a carbine don't look swell.

**Comment by Kuro020** [comment]
> > the blandest gun in the west, but it's ok
> 
> go harder on him, we both know the tan and middle section which looks like it was ripped from a carbine don't look swell.

but the actual g3 is like that so uhhhhhhhhh

**Comment by Tristan63** [comment]
totally NOT a g3a3

**Comment by davidchan13** [comment]
Interesting idea but why does the step between the T-12 and T-64 have less mag capacity than both of them?

**Comment by Hekzder** [comment]
I do agree with David that 25 rounds is incredibly low and doesn't make a lot of sense. Maybe 30 at least?

 I also feel with these kinds of stats it will just make the battle rifle even more obsolete? Though IMO that's more an issue of the battle rifle not being very good and less an issue of this being too power creep. If this does end up becoming a standard weapon I feel like a BR buff is even more overdue. Battle rifle will effectively fire almost half the speed while also being slower to handle for +2.5 damage, +5 penetration, +0.5 sunder, a somewhat larger magazine, and the bullets travel further. The bonuses you get for using the battle rifle, while there are a few, are incredibly small in nature and probably not worth it.

File: code/modules/projectiles/ammo_datums.dm [hunk 1]
```diff
@@ -500,6 +500,13 @@ datum/ammo/bullet/revolver/tp44
 	penetration = 10
 	sundering = 1
 
+/datum/ammo/bullet/rifle/heavy
+	name = "heavy rifle bullet"
+	hud_state = "hivelo"
+	damage = 30
+	penetration = 10
+	sundering = 1.5
+
```
Comment (SplinterGP) [inline code/modules/projectiles/ammo_datums.dm:509]: absurd stats for same firerate as the T-12

**Review by SplinterGP — CHANGES_REQUESTED** [review]

**Comment by Hekzder** [comment]
Should probably add a T-21 magazine box to the req operational vendor as well.

**Review by Kuro020 — CHANGES_REQUESTED** [review]
image
mob sprites are awful, fix

**Review by Kuro020 — APPROVED** [review]

**Comment by Hekzder** [comment]
Very cool, quite fun, and pretty well balanced as my first impressions. Seems like a pretty good addition.

**Comment by Al-1ce** [comment]
This is good. Very good. I'm a x*no main but I've tested around with this and it seems pretty balanced to me. Fills the niche that the T-37 should've.

**Comment by Tristan63** [comment]
> I do agree with David that 25 rounds is incredibly low and doesn't make a lot of sense. Maybe 30 at least?
> 
> I also feel with these kinds of stats it will just make the battle rifle even more obsolete? Though IMO that's more an issue of the battle rifle not being very good and less an issue of this being too power creep. If this does end up becoming a standard weapon I feel like a BR buff is even more overdue. Battle rifle will effectively fire almost half the speed while also being slower to handle for +2.5 damage, +5 penetration, +0.5 sunder, a somewhat larger magazine, and the bullets travel further. The bonuses you get for using the battle rifle, while there are a few, are incredibly small in nature and probably not worth it at that point.

battle rifle has always been obsolete

File: code/modules/projectiles/ammo_datums.dm (hunk 1 above)
Comment (SplinterGP) [reply code/modules/projectiles/ammo_datums.dm:509]: i say still reduce this sunder to like 1 or 1.25

**Review by SplinterGP — CHANGES_REQUESTED** [review]

**Review by David-Stormwell — APPROVED** [review]
Should be fine as is, feel free to enforce that sunder change though other things deal more sunder, so it really shouldn't matter.

**Comment by lbnesquik** [comment]
I think we are done here. I'm up for a full merge.

**Comment by AnCopper** [comment]
MJP is incapable of creating anything other than old war era guns.

**Pull request merged**